Fix blame

laksono opened the pull request:
- Update blame-shifting so it doesn't break the summary view. The latter needs to show the idleness, while the original blame-shifting remove this.
- Fix a bug when reading trace.db. This happens when we don't have enough buffer in a page.

laksono committed:
Draft fix blame-shifting

laksono committed:
Fix cpu-gpu blame shifting

laksono committed:
Clean-up blame shifting

laksono committed:
cod clean-up

laksono committed:
Fix byte buffer when reading trace.db due to insufficient buffer

Generalize the solution to read from LargeByteBuffer some bytes when
there is no enough buffer.
For instance, when a page is 1024 bytes, but we want to read 8 bytes
from index 1022, then we have to combine 2 bytes from the current page
and 6 bytes from the next page.

This commit fix for Long, Double, Float and Short

laksono committed:
Convert node index to hexadecimal

dejangrubisic reviewed (commented):
I think we we can implement blame-shift this way too for now. Once we have more information about utilization of streams and which tread launched what stream, we can implement more complex blame-shifting

laksono committed:
Fix percent column in blame-shift view for MacOS

We shouldn't call pack() method on Mac. It will try to squeeze the
column based on the width of the column title.

Minor update:
- Fix assertion in LargeByteBuffer
- Use the standard tooltip delay preference, not a constant
